src/pages/Vaults/Vault/RequestRedeemModal/index.tsx

```diff
@@ -60,6 +62,8 @@ const RequestRedeemModal = ({ onClose, open, collateralToken, vaultAddress, lock
       const vaultId = newVaultId(window.bridge.api, vaultAddress, collateralToken, WRAPPED_TOKEN);
       await window.bridge.redeem.request(amountPolkaBtc, data[BTC_ADDRESS], vaultId);
 
+      queryClient.invalidateQueries(['vaultsOverview', vaultAddress, collateralToken.ticker]);
```

Comment:
might not be necessary, but should not be a problem refetching after redeem.

Reply:
Looks good to me - sensible to refetch/invalidate after any action

Reply:
Not at all important now, but might be worth considering whether we get into the habit of returning a invalidate function from our hooks, so that we don't need to update query keys manually if we ever change them?